cirq-google/cirq_google/calibration/workflow.py

```diff
@@ -834,6 +853,42 @@ def make_zeta_chi_gamma_compensation_for_moments(
         The moment to calibration mapping is updated for the new circuit so that successive
         calibrations could be applied.
     """
+
+    if isinstance(circuit, Circuit):
```

Comment:
factor this out into a helper method?

Reply:
Done, I wanted to send out this PR quickly so it wasn't really clean.